lib/JSON/Validator.pm

```diff
@@ -114,6 +114,14 @@ sub bundle {
   return $bundle;
 }
 
+# $self->errors_conf( complex_as_list => 1 );
+sub errors_conf {
```

Comment:
Why is this a method and not a simple attribute like `has errors_conf => sub { +{} };`?

Reply:
The idea was just to keep the door open for other possible options like in "coerce".

Reply:
coerce() is a method for back compat. No need for it here when it's a new attribute.